### PR TITLE
Adding Peer Review

### PR DESCRIPTION
**Pull request by dHelmgren**
Added this within the context of the project so that students can read it on their machine when they finish the project.

### CODE REVIEW

File: .peer-review.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+
+When you start working in industry, whether for a large company or a small start-up, there is a pretty good chance that at some point you will be asked to review code written by your peers before it is pushed to your codebase. 
+
+There are a lot of good reasons why this practice exists: it helps us reduce errors, build understanding of parts of the codebase that we don't regularly interact with, and developing trust among team members.
+
+GitHub, as a version control platform, gives us quite a few tools to help with the process of doing a code review, including comments, pull requests, bug reports and approvals.
+
+## Learning Goals
+
+By the end of this activity, you will be able to
```
Comment (kaidamasaki) [inline .peer-review.md]: Consider:
```suggestion
By the end of this activity, you will be able to:
```

or

```suggestion
By the end of this activity, you will be able to...
```

File: .peer-review.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+- Explain the value of doing peer code reviews
+- Use GitHub's tools to engage with the code of your peers
+
+## Getting Started
+
+Whenever we ask you to turn in homework, we have you make a Pull Request, or PR. A PR is a way of asking for other contributors on the project to look at your code and check it for bugs or errors. If you navigate to the repository you cloned from and look at the top bar for the repo, you should see a button labeled Pull Request, see below. Click that button.
```
Comment (kaidamasaki) [inline .peer-review.md]:
```suggestion
Whenever we ask you to turn in homework, we have you make a Pull Request, or "PR". A PR is a way of asking for other contributors on the project to look at your code and check it for bugs or errors. If you navigate to the repository you cloned from and look at the top bar for the repo, you should see a button labeled "Pull Request", see below. Click that button.
```

File: .peer-review.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+![A GitHub Repo main page. The Pull Request button is thrid from the left, after Code and Issues](.images/pull-request-highlight.png "A screenshot of a GitHub Repo main page. The Pull Request button is thrid from the left, after Code and Issues")
```
Comment (kaidamasaki) [inline .peer-review.md:21]: I really like the detailed `alt` text.

File: .peer-review.md [hunk 4]
```diff
@@ -1 +1,59 @@
...
+
+![An example of a student's Startrly pull request. The Files Changed tab is on the second line of tabs, and is the fourth from the left.](.images/on-the-branch.png "An example of a student's Startrly pull request. The Files Changed tab is on the second line of tabs, and is the fourth from the left.")
+
+From here, we can hover over one of the line numbers on the left hand side to insert a comment. Clicking the blue backed plus sign that appears will open a comment dialog. 
+
+![This image shows a page of code on GitHub. A blue backed plus sign appears on the left hand side of one of the lines.](.images/hover-to-comment.png "This image shows a page of code on GitHub. A blue backed plus sign appears on the left hand side of one of the lines.")
+
+Once you have filled in your first comment, you can click the button in the bottom right corner of the dialog labeled Start A Review. Note that the the four lines above the comment will be visible in previews of the comment, so make sure that the relevant code is at least partially in that window.
```
Comment (kaidamasaki) [inline .peer-review.md]:
```suggestion
Once you have filled in your first comment, you can click the button in the bottom right corner of the dialog labeled "Start A Review". Note that the the four lines above the comment will be visible in previews of the comment, so make sure that the relevant code is at least partially in that window.
```

**Review by kaidamasaki — APPROVED** [review]
Aside from some small things it looks good!

**Comment by dHelmgren** [comment]
[Startrly Peer Review Activity](https://trello.com/c/s8m2rzPl/1276-startrly-peer-review-activity)

**Review by tildeee — APPROVED** [review]
Looks amaaaazing!

Could we have a resource at the bottom to link to another resource we've already written? https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/03-leadership-and-inclusion/pair-feedback-and-code-reviews.md#how-to-give-a-code-review

Otherwise I think it looks greeeeattttt!!!!

**Pull request merged**